hashtree_abi.nim

```diff
@@ -0,0 +1,29 @@
+# hashtree nim bindings
+# Copyright (c) 2024 Status Research & Development GmbH
+# Licensed and distributed under either of
+#   * MIT license (license terms in the root directory or at https://opensource.org/licenses/MIT).
+#   * Apache v2 license (license terms in the root directory or at https://www.apache.org/licenses/LICENSE-2.0).
+# at your option. This file may not be copied, modified, or distributed except according to those terms.
+
```

Comment:
`{.push raises: [].}` here maybe a bit more consistent than adding it to each proc/func

Reply:
a300d24da3482bc1a548467654b6bc1b98966fbc - on imported functions, push doesn't work too well, but we can avoid some copypaste at least